lib/fontist/errors.rb

```diff
@@ -9,10 +10,26 @@ class LicensingError < GeneralError; end
     class ManifestCouldNotBeFoundError < GeneralError; end
     class ManifestCouldNotBeReadError < GeneralError; end
     class MissingAttributeError < GeneralError; end
-    class MissingFontError < GeneralError; end
-    class NonSupportedFontError < GeneralError; end
     class TamperedFileError < GeneralError; end
     class TimeoutError < GeneralError; end
     class UnknownFontTypeError < GeneralError; end
+
+    class FontError < GeneralError
+      attr_reader :font, :style
+
+      def initialize(msg, font = nil, style = nil)
+        @font = font
+        @style = style
+
+        super(msg)
+      end
+
+      def name
+        "#{@font} #{@style}"
```

Comment:
The "font" and "style" names will be confusing if they contain spaces, please separate them with a pattern that is notable, "Font name: 'xxx'; Style: 'xxx'"

Reply:
Done